### ISSUE TITLE
Learn the platform with svelte

### ISSUE THREAD

**Issue by paoloricciuti** [lnug/speakers#205]
Svelte is a frontend framework but despite that thanks to svelte-kit and the general philosophy behind svelte it's a great tool to get started in web dev because while learning svelte you can easily learn how the web platform works, and now with svelte-kit you can learn node too!

I'm Paolo Ricciuti, i work as a full stack developer at Main matter and I'm a svelte lover (and also an official svelte ambassador).

- [x] I confirm that I will abide by the [Code of Conduct](https://lnug.org/code-of-conduct.html)

- [x] I confirm my talk is 20 mins long in total.

**Comment by admataz** [comment]
Hey @paoloricciuti  - I'm a big fan of Svelte (and previously Sapper)  - and would love to hear more about the recent svelte-kit developments. 

Would you be talking about _how we as node developers can use_ svelte-kit effectively - or will you be talking more about _how svelte-kit is authored_ with regards to node.js/ javascript/ typescript  - I think both angles would be of interest to LNUG attendees. 

How are you for a talk in October?

**Comment by paoloricciuti** [comment]
> Hey @paoloricciuti - I'm a big fan of Svelte (and previously Sapper) - and would love to hear more about the recent svelte-kit developments.
> 
> Would you be talking about _how we as node developers can use_ svelte-kit effectively - or will you be talking more about _how svelte-kit is authored_ with regards to node.js/ javascript/ typescript - I think both angles would be of interest to LNUG attendees.
> 
> How are you for a talk in October?

My idea was to start with a simple html file and show it can be used to build some simple Todo, than literally copy and paste all of that in the svelte repl to see how svelte is close to raw html/css/JavaScript and how it show you warnings for a11y etc. From there how to move to full stack with sveltekit and see how there you would use native request and response and how you can use fs modules and other node.js stuff

**Comment by paoloricciuti** [comment]
> > Hey @paoloricciuti - I'm a big fan of Svelte (and previously Sapper) - and would love to hear more about the recent svelte-kit developments.
> > Would you be talking about _how we as node developers can use_ svelte-kit effectively - or will you be talking more about _how svelte-kit is authored_ with regards to node.js/ javascript/ typescript - I think both angles would be of interest to LNUG attendees.
> > How are you for a talk in October?
> 
> My idea was to start with a simple html file and show it can be used to build some simple Todo, than literally copy and paste all of that in the svelte repl to see how svelte is close to raw html/css/JavaScript and how it show you warnings for a11y etc. From there how to move to full stack with sveltekit and see how there you would use native request and response and how you can use fs modules and other node.js stuff

When would the October meetup be? I think I can make it work. Is it ok to attend virtually?

**Comment by admataz** [comment]
Are you going to be in London at any point? We are an in person meet up primarily. We have found remote talks don't really work very well.

**Comment by admataz** [comment]
It will be the last week of October - details to be confirmed. Next one would be late November.

**Comment by paoloricciuti** [comment]
I might be able to organize to come to London, at this point tho it might be better to also sync with the svelte London meetup so that I can catch two pigeon with a stone. Are you flexible with dates?

**Comment by paoloricciuti** [comment]
Scott confirmed that the svelte meetup should be the last week of October too...they are still finding a venue so it's not set in stone yet but I think we can coordinate

**Comment by admataz** [comment]
Awesome! I'll lock in a date for October and we can work from there.

**Comment by paoloricciuti** [comment]
@admataz svelte London meetup is the 25th...can you fix a date near this (I don't know how much flexibility you have)

**Comment by admataz** [comment]
Hey @paoloricciuti  - we have secured Tuesday 24th October for our next meetup. Can you make it?

**Comment by paoloricciuti** [comment]
Yeah It's ok...I'll book my plane next week

**Comment by paoloricciuti** [comment]
Hey @admataz how strict is the 20 min rule? How far can i go over that? 😅

**Comment by admataz** [comment]
Hey @paoloricciuti we can be flexible and plan a bit. We just want to avoid rambling overruns that affect time available for later speakers and people that need to go home. There's nothing worse than when people have to leave between talks. (Has happened) 

I think up to 30mins would be ok. Is that long enough?

**Comment by paoloricciuti** [comment]
> Hey @paoloricciuti we can be flexible and plan a bit. We just want to avoid rambling overruns that affect time available for later speakers and people that need to go home. There's nothing worse than when people have to leave between talks. (Has happened)
> 
> I think up to 30mins would be ok. Is that long enough?

Yeah absolutely: I was asking because my original idea was taking a bit longer and I can shorten it but it would be a bit worst. I will keep it under 30min

**Comment by admataz** [comment]
Hey @paoloricciuti   - hope you're all good for tomorrow?.

I will try get to the venue as soon after 5pm as possible to start making things ready. Makers have a zoom - based video set up for broadcasting to their projectors- let me know if you can't run that and we will make a plan. 

Also Let me know if you need anything special - otherwise we will just plug your laptop in and go... looking forward to seeing you there.

**Comment by paoloricciuti** [comment]
> Hey @paoloricciuti - hope you're all good for tomorrow?.
> 
> I will try get to the venue as soon after 5pm as possible to start making things ready. Makers have a zoom - based video set up for broadcasting to their projectors- let me know if you can't run that and we will make a plan.
> 
> Also Let me know if you need anything special - otherwise we will just plug your laptop in and go... looking forward to seeing you there.

I should be able to connect pretty easily, I'll try to be there as soon as I can so that if we have problems we can have time to sort them out.

**Comment by paoloricciuti** [comment]
@admataz I should be out there but I don't know where the entrance is

**Comment by admataz** [comment]
HI @paoloricciuti  - I'm sorry - the video for this was never made available after Pusher closed shop on us. Hopefully see you again at a future LNUG - and maybe  you can bring us a latest update from the world of Svelte in a pt 2 of your talk 😄 ?

**Comment by paoloricciuti** [comment]
@admataz would be happy to propose another talk if you are available...when would the next meetup be?

**Comment by admataz** [comment]
Hi @paoloricciuti  - we are planning the next 3 months now - September is looking like a full programme already.  How are you for 22 (or maybe 29th) October or 26 November ?

**Comment by admataz** [comment]
We're hoping to get back to a monthly rhythm in the new year...
